Content.Server/Store/Conditions/StoreLimitedStockCondition.cs

```diff
@@ -0,0 +1,17 @@
+using Content.Shared.Store;
+
+namespace Content.Server.Store.Conditions;
+
+/// <summary>
+/// Only allows a listing to be purchased a certain amount of times.
+/// </summary>
+public sealed class StoreLimitedStockCondition : ListingCondition
```

Comment:
Fields and functions of this need doc comments. Remember `<inheritdoc/>` exists for overrides.